### DISCUSSION TITLE
Backup full Iphone15pro

### DISCUSSION THREAD

**Discussion by Billy7899** [icloud-photos-downloader/icloud_photos_downloader#940] [Q&A]
Hi, i am searching for a full backup
of the icloud with folders an so on.

I downloaded this now from the apple privacy, but to backup 200GB it takes a long time and there are no folders and so on....

Now i found this project and it looks very well....great !

My question, can i download the 200GB without any problems and the right folder structure ?
There are many different meanings....
Thx
Billy

**Comment by AndreyNikiforov** [comment]
you can use `-dry-run` to see what `icloudpd` would do for you and then decide it that matches your plans. see [doc](https://icloud-photos-downloader.github.io/icloud_photos_downloader/install.html) for more nuances
